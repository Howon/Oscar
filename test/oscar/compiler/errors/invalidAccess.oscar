def main() => unit = {
  list<int> l1 = list<int>[1, 2, 3, 4, 5]; 
  int x = l1["0"];
}