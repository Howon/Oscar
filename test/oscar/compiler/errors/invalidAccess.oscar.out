Invalid access types List_t lt, Int_t cannot be accessed by String_t